.rubocop.yml

```diff
@@ -1,4 +1,87 @@
 inherit_from: .rubocop_todo.yml
 
+# Turn on RSpec cops
+require: rubocop-rspec
+
+AllCops:
+  DisplayCopNames: true
+  Include:
+    - Rakefile
+    - config.ru
+    - lib/**/*.rake
+  Exclude:
+    - 'Gemfile.lock'
+    - '**/*.md'
+    - 'bin/**/*'
+    - 'config/**/*.yml'
+    - 'db/**/*'
+    - 'script/**/*'
+    - 'vendor/**/*'
+
+# Turn on Rails cops
+Rails:
+  Enabled: true
+
+Bundler/OrderedGems:
+  Exclude:
+    - 'Gemfile'
+
+# --- Layout ---
+
+Layout/EmptyLinesAroundBlockBody:
+  Enabled: false # we don't need to care/æ
+
+Layout/IndentationConsistency:
+  EnforcedStyle: rails
+
+Layout/SpaceInsideBrackets:
+  Exclude:
+    - 'config/environments/production.rb' # that's how Rails generates it
+
+# --- Metrics ---
+
+# because this isn't 1994
+Metrics/LineLength:
+  Max: 120
+  Exclude:
+    - 'app/views/checksum_records/_checksum_record.json.jbuilder'
+
+Metrics/MethodLength:
+  Max: 15
```

Comment:
In the Preservation Catalog, the method length max is set to 20. Was there any particular reason why you set it to 15 here? It seems a little low to me, maybe even 25 would be better.

Reply:
i just picked a number.  I think the rubocop problem was for 11 lines.  We can bump it up higher later.   It's not bad to be reminded to look at a method to see if it sh/could be shorter.